### ISSUE TITLE
Convert 4104 Classic form to using XML

### ISSUE THREAD

**Issue by MHaggis** [splunk/security_content#2039]
Reported here https://splunk-usergroups.slack.com/archives/C78NT6CQ7/p1645207975243339

 ESCU - Detect Empire with PowerShell Script Block Logging - Rule I think is using the classic form of the 4104s. I say I think because it certainly isn't matching the XML format I'm seeing.
```
Your version                 |                     4104 field name
OpCode                         |                      Opcode
ComputerName              |                      Computer
User                             |                      UserID  (it's a SID, not a user name)
Message                       |                      ScriptBlockText (edited) 
```

I only bring it up, as it uses the powershell macro, which is set to pull source of classic or xml, so one part is dynamic but the field names are not.

here's a search that looks for only a few of the common Windows classic fields in correlation searches. Can see that ESCU has 30 some odd searches that need "updating" to conform to XML.
```
 | rest splunk_server=local count=0 /servicesNS/-/-/saved/searches
| search action.notable.param.rule_title=*
| where match(qualifiedSearch, "Process_Name") OR match(qualifiedSearch, "Account_Name") or match(qualifiedSearch, "Message=")
| search NOT qualifiedSearch IN ("*okta*")
| table title eai:acl.app qualifiedSearch
```

**Comment by MHaggis** [comment]
- ESCU - Delete ShadowCopy With PowerShell - Rule
- ESCU - Detect Empire with PowerShell Script Block Logging - Rule
- ESCU - Detect Mimikatz Via PowerShell And EventCode 4703 - Rule
- ESCU - Enumerate Users Local Group Using Telegram - Rule
- ESCU - GetWmiObject Ds Computer with PowerShell Script Block - Rule
- ESCU - GetWmiObject Ds Group with PowerShell Script Block - Rule
- ESCU - GetWmiObject User Account with PowerShell Script Block - Rule
- ESCU - Interactive Session on Remote Endpoint with PowerShell - Rule
- ESCU - Known Services Killed by Ransomware - Rule
- ESCU - Multiple Disabled Users Failing To Authenticate From Host Using Kerberos - Rule
- ESCU - Multiple Invalid Users Failing To Authenticate From Host Using Kerberos - Rule
- ESCU - Multiple Users Attempting To Authenticate Using Explicit Credentials - Rule
- ESCU - Multiple Users Failing To Authenticate From Host Using Kerberos - Rule
- ESCU - Multiple Users Failing To Authenticate From Process - Rule
- ESCU - Multiple Users Remotely Failing To Authenticate From Host - Rule
- ESCU - PetitPotam Network Share Access Request - Rule
- ESCU - PetitPotam Suspicious Kerberos TGT Request - Rule
- ESCU - Powershell Fileless Process Injection via GetProcAddress - Rule
- ESCU - Powershell Fileless Script Contains Base64 Encoded Content - Rule
- ESCU - Recon Using WMI Class - Rule
- ESCU - Remote Process Instantiation via DCOM and PowerShell Script Block - Rule
- ESCU - Remote Process Instantiation via WMI and PowerShell Script Block - Rule
- ESCU - Remote Process Instantiation via WinRM and PowerShell Script Block - Rule
- ESCU - ServicePrincipalNames Discovery with PowerShell - Rule
- ESCU - Short Lived Scheduled Task - Rule
- ESCU - Suspicious Computer Account Name Change - Rule
- ESCU - Suspicious Kerberos Service Ticket Request - Rule
- ESCU - Suspicious Ticket Granting Ticket Request - Rule
- ESCU - Unloading AMSI via Reflection - Rule
- ESCU - WMI Recon Running Process Or Services - Rule

**Comment by TheLawsOfChaos** [comment]
This should probably be revisited/reopened.

**Comment by MHaggis** [comment]
Merged in today and will go out under 3.45.0.
Thank you!
